lib/rucio/core/transfer.py

```diff
@@ -932,7 +932,7 @@ class _ClassNameDescriptor:
         """
         Automatically set the external_name of the strategy to the class name.
         """
-        def __get__(self, obj, objtype=None):
+        def __get__(self, obj: object, objtype: Optional[Any] = None) -> str:
```

Comment:
```suggestion
        def __get__(self, obj: object, objtype: Any = None) -> str:
```

`Any` allows for `None` as well, so `Optional[Any]` is not necessary

Reply:
Does the CI ignore `ANN401`? My flake8 does not like naked `Any` annotations

![image](https://github.com/user-attachments/assets/e47b71de-e8d6-4f72-bd43-7407f77c6ebb)

Reply:
`ANN` is not enabled: https://github.com/rucio/rucio/blob/e6097bd2ee4efb1ade8b4e262a28efdda6fb96a2/pyproject.toml#L56-L87 - it might be your local Ruff install that complains about it, but the CI should be alright (we use `Any` in quite a few other places)